src/library/mod.rs

```diff
@@ -392,6 +392,25 @@ impl Library2 {
         }
     }
 
+    /// checks if the current processor is a lazy processor
+    /// or not
+    ///
+    /// lazy processor = processor which needs to be resolved after
+    /// interpretation
```

Comment:
This should go to glossary and comments should simply contain `/// fpm.dev/glossary/#lazy-processor`.